reconcile/utils/terrascript/cloudflare_resources.py

```diff
@@ -177,7 +177,7 @@ def populate(self) -> list[Union[Resource, Output]]:
             resources.append(cloudflare_argo(self._spec.identifier, **argo_values))
 
         for record in zone_records:
-            identifier = safe_resource_id(record.get("name"))
+            identifier = safe_resource_id(record.pop("identifier"))
```

Comment:
This should probably be breaking a test. Any thoughts about modifications to a test in `reconcile/test/test_utils_terrascript_cloudflare_client.py` that would have caught this change?

Reply:
Yeah I'll update `test_create_cloudflare_resources_terraform_json`